extensions/sql-migration/src/api/azure.ts

```diff
@@ -238,53 +238,53 @@ export async function getAvailableSqlDatabaseServers(account: azdata.Account, su
 	const api = await getAzureCoreAPI();
 	const path = encodeURI(`/subscriptions/${subscription.id}/providers/Microsoft.Sql/servers?api-version=${SQL_SQLDB_API_VERSION}`);
 	const host = api.getProviderMetadataForAccount(account).settings.armResource?.endpoint;
-	const response = await api.makeAzureRestRequest(account, subscription, path, azurecore.HttpRequestMethod.GET, undefined, true, host);
+	const response = await api.makeAzureRestRequest<any>(account, subscription, path, azurecore.HttpRequestMethod.GET, undefined, true, host);
```

Comment:
No, this is just a typings update which don't even exist at runtime. It will cause compilation failures - but that's fine.

Reply:
The same file has caused issues in the past, and changed had to be reverted from here so that's why wanted to double check. https://github.com/microsoft/azuredatastudio/pull/22761/files#diff-4081e8f797712d175859ec2168fe2d2dc901c22daad99608fb7bb77fee2ac3e7

Reply:
If an existing extension version can continue to work with these azurecore extension changes, then I have no concerns.

Reply:
That has nothing to do with the signature/typings - but rather the fact that the object we were returning had its property names changed (from data to body). And stuff like that is why it's critically important to have actual typings for these things instead of just `any` everywhere - otherwise changes to the objects can easily slip by without being caught by the compiler. 

Nothing like that is being done here - this is all just typings updates which tell the compiler what we expect the different values to be. But all that information disappears at runtime, at that point it's just the actual objects with their properties (whatever they might be named).